jenkins/job-configs/jenkins-hourly-maintenance.yaml

```diff
@@ -0,0 +1,64 @@
+- job:
+    name: 'jenkins-hourly-maintenance'
+    concurrent: true
+    description: 'Clean docker images. Test owner: spxtr.'
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+    builders:
+        - shell: |
+            # Select only exited or dead containers that weren't created this hour.
+            # A sort of work-around for https://github.com/kubernetes/kubernetes/issues/24661
+            containers_to_remove=($(docker ps -a -f 'status=exited' -f 'status=dead' --format='{{.CreatedAt}}\t{{.ID}}' | grep -v ^$(date +'%Y-%m-%d %H') | cut -f2 || true))
```

Comment:
can we make sure this job runs some number of minutes after the hour, so that it's unlikely to cause kubernetes/kubernetes#24661 to recur?